.github/workflows/release.yaml

```diff
@@ -0,0 +1,45 @@
+# Prefect Operator release workflow.
+name: Release
+
+on:
+  release:
+    types: [published]
+
+permissions: {}
+
+jobs:
+  build-and-upload-manifests:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: jdx/mise-action@v2
+      - name: Build and upload manifests
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd config/manager && kustomize edit set image controller=${{ github.ref_name }}
+          kustomize build config/crds > prefect-crds.yaml
+          kustomize build config/default > prefect.yaml
+          gh release upload ${{github.event.release.tag_name}} prefect-crds.yaml prefect.yaml
+
+  build-and-push-docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
Will need to populate these.

Reply:
@mitchnielsen we can add them in Terraform.  i'll DM you

Reply:
https://github.com/PrefectHQ/platform/pull/7409